src/header.rs

```diff
@@ -64,6 +65,11 @@ pub struct Header {
     #[serde(skip_serializing_if = "Option::is_none")]
     #[serde(rename = "x5t#S256")]
     pub x5t_s256: Option<String>,
+
+    /// Any additional headers
```

Comment:
It would be good to give more info here - what it is, what it's for, what it looks like in serialized form.